manifests/config.pp

```diff
@@ -32,16 +32,7 @@
     mode   => '0755',
   }
 
-  foreman_config_entry { 'pulp_client_cert':
-    value          => $::certs::pulp_client::client_cert,
-    ignore_missing => false,
-    require        => [Exec['foreman-rake-db:seed'], Class['::certs::pulp_client']],
-  }
-
-  foreman_config_entry { 'pulp_client_key':
-    value          => $::certs::pulp_client::client_key,
-    ignore_missing => false,
-    require        => [Exec['foreman-rake-db:seed'], Class['::certs::pulp_client']],
-  }
+  class { '::certs::pulp_client': }
```

Comment:
Might as well move this into the pulp_client_config class, and I might consider just using the namespace, e.g.

```
config/pulp_client.pp
```

```
katello::config::pulp_client
```